database/javascript/2016/10695.yaml

```diff
@@ -0,0 +1,27 @@
+---
+cve: 2016-10695
+title: CVE in sqlite3
+description: >
+    The npm-test-sqlite3-trunk module provides asynchronous, non-blocking SQLite3 bindings. npm-test-sqlite3-trunk downloads binary resources over HTTP, which leaves it vulnerable to MITM attacks. It may be possible to cause remote code execution (RCE) by swapping out the requested resources with an attacker controlled copy if the attacker is on the network or positioned in between the user and the remote server.
+cvss_v2: 9.3
+references:
+    - https://nodesecurity.io/advisories/297
+
+affected:
+    - name: sqlite3
```

Comment:
The name should be `npm-test-sqlite3-trunk` and not just `sqlite3`.

Reply:
Good catch ;)